lib/schema/ports.go

```diff
@@ -0,0 +1,76 @@
+/*
+Copyright 2020 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package schema
+
+// DefaultPortRanges defines the list of default ports for the cluster
+var DefaultPortRanges = PortRanges{
+	Kubernetes: []PortRange{
+		{Protocol: "tcp", From: 2379, To: 2380, Description: "etcd"},
+		{Protocol: "tcp", From: 6443, To: 6443, Description: "kubernetes API server"},
+		{Protocol: "tcp", From: 10248, To: 10255, Description: "kubernetes internal services range"},
+		{Protocol: "tcp", From: 7496, To: 7496, Description: "serf (health check agents) peer to peer"},
+		{Protocol: "tcp", From: 7373, To: 7373, Description: "serf (health check agents) peer to peer"},
+	},
+	Installer: []PortRange{
+		{Protocol: "tcp", From: 61009, To: 61010, Description: "installer ports"},
+		{Protocol: "tcp", From: 61022, To: 61025, Description: "installer ports"},
+		{Protocol: "tcp", From: 61080, To: 61080, Description: "installer ports"},
```

Comment:
Is this actually used anywhere?

Reply:
Yes, [it is](https://github.com/gravitational/gravity/blob/dmitri/6.1.x/3224-selinux-container/vendor/github.com/gravitational/teleport/lib/config/configuration.go#L464) - let me know if that's also bogus.